presto/testing/integration_tests/tpch_test.py

```diff
@@ -16,11 +16,11 @@
 
 from . import test_utils
 from .common_fixtures import presto_cursor, setup_and_teardown
-from ..common.fixtures import tpch_queries
+from ..common.fixtures import tpch_queries, get_scale_factor
 
 BENCHMARK_TYPE = "tpch"
 
 
-@pytest.mark.usefixtures("setup_and_teardown")
+@pytest.mark.usefixtures("setup_and_teardown", "get_scale_factor")
```

Comment:
I don't think this update is needed, since `tpch_queries` already depends on `get_scale_factor`?